vendor/brave-ios/Ads/BATBraveAds.mm

```diff
@@ -878,6 +878,13 @@ - (void)downloadUserModelForId:(NSString *)id completion:(void (^)(BOOL success,
   }];
 }
 
+- (void)getBrowsingHistory:(const int)max_count
+                   forDays:(const int)days_ago
+                  callback:(ads::GetBrowsingHistoryCallback)callback {
+  // Not implemented for iOS
```

Comment:
nit: 'To be implemented` with a link to a `brave-ios` issue so that the work can be prioritized